src/metadataEditor.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
+#include "metadataEditor.hpp"
+
+// Constructor
+metadataEditor::metadataEditor(int xx = 23){
```

Comment:
I see the CI caught another error that I didn't see, default arguments for functions should go in the `.hpp` file, not the `.cpp` file.